pages/radar.md

```diff
@@ -7,6 +7,9 @@ permalink: /radar.html
 
 [/home](/)
 
+## [Linux turns 30](https://twitter.com/linuxfoundation/status/1430539222142885898)
+25-August-2021: 30 years ago today Linus Torvalds introduced Linux to the world. Since then, Linux has become the leading operating system widely used in many applications, and in some, this is a de facto monopoly - say, [100% of supercomputers run Linux](https://itsfoss.com/linux-runs-top-supercomputers/). The operating system is constantly improving, new functions appear every release. Such active development has become possible because open licensing model. More than three-quarters of the new changes are committed by corporations. For example, Huawei Technologies [made 6.5% of all changes](https://lwn.net/Articles/860989/) for the latest stable kernel. What's the profit? The use in company products of this large collaborated development brings far more revenue than the cost of changes was made. The development of such a fully proprietary system is much more expensive and consequently would increase the final cost of products. It is clear that an open model of development is more competitive. The world runs on open source.
```

Comment:
25-August-2021: 30 years ago today Linus Torvalds introduced Linux to the world. Since then, Linux has become the leading operating system widely used in many applications having a de facto monopoly in some of them - [100% of supercomputers run Linux](https://itsfoss.com/linux-runs-top-supercomputers/). The operating system is constantly improving, and new functions appear every release. Such active development has become possible because of the open licensing model. More than three-quarters of the new changes are committed by corporations. For example, Huawei Technologies [made 6.5% of all changes](https://lwn.net/Articles/860989/) for the latest stable kernel. What's the profit? This extensive collaborated development in company products brings far more revenue than the cost of implementing changes. Developing such a fully proprietary system is much more expensive and would consequently increase the final price of products. It is clear that an open model of development is more competitive. The world runs on open source.

Reply:
Fixed